model/host/host.go

```diff
@@ -31,7 +31,8 @@ type Host struct {
 	Zone    string `bson:"zone" json:"zone"`
 
 	// true if the host has been set up properly
-	Provisioned bool `bson:"provisioned" json:"provisioned"`
+	Provisioned       bool `bson:"provisioned" json:"provisioned"`
```

Comment:
Don't think this is used at all

Reply:
Isn't it used in functions like `MarkAsProvisioned()`?

Reply:
Isn't it used in functions like `MarkAsProvisioned()`?